README.md

```diff
@@ -1,2 +1,85 @@
 # probby
+
 Probby is the friendly Exercism problem-specs robot.
+
+## Usage
+
+### `problem-specifications` repo
+
+1. Create a machine user with write access to all tracks that you want to push to.
+2. Create a Personal Access Token (★) with the `public_repo` scope for the machine user.
+3. Create an environment called `probby_environment`
+    1. Add up to 6 trusted reviewers as _Required reviewers_. **Make sure to hit _Save protection rules_!**
+    2. Create an **environment** secret `PROBY_TOKEN` with the PAT (★) from above as content.
+4. Add the workflow below as `.github/workflows/probby.yml`.
+5. Change the value of `track-repos` to suit your needs.
+
+```yaml
+name: Probby
+
+on:
+    push:
+        branches:
+            - 'probby-tests'
+
+jobs:
+    parse-push:
+        name: Parse push event and prepare payload
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            - uses: SaschaMann/probby/packages/parse-push@v1
+              id: parse-push
+
+            - uses: actions/upload-artifact@v2
+              with:
+                  name: payload
```

Comment:
Cool to see this idea progressing already 👌.
I don't know much about this stuff but the following question occurs to me here: Will this name clash of there are several problem-spec PRs merged at the same/similar time?

Reply:
They're attached to workflow runs/builds rather than being repo-wide, so that situation won't cause issues.